contracts/src/access/control.rs

```diff
@@ -522,7 +522,7 @@ mod tests {
     }
 
     #[motsu::test]
-    fn the_new_admin_can_grant_roles(contract: AccessControl) {
+    fn role_admin_change_success_with_new_grant(contract: AccessControl) {
```

Comment:
```suggestion
    fn grant_role_succeeds_when_called_by_new_admin(contract: AccessControl) {
```
- `grant_role` is actually being tested here
- related to the previous comment, make sure that all test case names follow the expected convention